src/main/resources/templates/journals/edit.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/extras/spring-security">
+<head>
+    <meta charset="UTF-8"/>
+    <title>Edit a Post</title>
+    <th:block th:replace="partials/elements :: head(title='FilmSage Index')"></th:block>
```

Comment:
there is already a head tag inside the partial template brought in on this line. delete everything from line 3 to line 7 except for this line and change the tag here from th:block to head. change head(title='FilmSage Index') to head(title='Edit Journal')